.circleci/config.yml

```diff
@@ -25,6 +25,18 @@ workflows:
       - package_storage_plus
       - lint
       - wasm-build
+      - benchmarking:
+          requires:
+            - package_storage_plus
+          filters:
+            branches:
+              only:
+                # Long living branches
+                - main
+                - /^[0-9]+\.[0-9]+$/
+                # 👇Add your branch here if benchmarking matters to your work
+                - benchmarking
+                - signed-int-key-bench
```

Comment:
:+1: for filtering, although I would still cut that list a little bit shorter. 
Maybe remove release tag branches, since they are basically always directly derive from `main` anyway?